tests/cases/compiler/maxNodeModuleJsDepthDefaultsToZero.ts

```diff
@@ -4,7 +4,7 @@
 // @traceResolution: true
 // @noEmit: true
 
-// @filename: c:/root/tsconfig.json
+// @filename: tsconfig.json
```

Comment:
You need to include the `/` in front or else your own directory will show up in the trace.